src/kili/services/export/format/base.py

```diff
@@ -146,6 +147,10 @@ def export_project(
         """
         self._check_arguments_compatibility()
         self.logger.warning("Fetching assets...")
+        if self.download_media:
+            count = self.kili.count_assets(project_id=self.project_id)
+            if count > self.THRESHOLD_WARN_MANY_ASSETS:
+                self.logger.warning(f"Downloading {count} assets. This might take a while.")
```

Comment:
imho it would be worth if to put this verification in the "client front-end layer", and add a front-end specific message on how to disable the downloading.
200 is low imho, you can put 1000 (even if this is content size specific of course)
Also, you can use `warnings.warn` to let the user specify if he/she wants to disable the specified warning.

Reply:
in what module would you put the verification exactly?

Reply:
in `kili.export_labels` and `kili.cli.project.export`

Reply:
I put the warning in `services.export_labels` since this method is called from several places (CLI, QueriesLabel and Project). It's easier and better to keep it in one single place, I think.